### PR TITLE
Afsaneh/exchange rate

### PR DESCRIPTION
**Pull request by afsaneh-rezaeinejad-deriv**

### CODE REVIEW

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js [hunk 1]
```diff
@@ -1,16 +1,50 @@
...
+        cy.log(finalRate)
+        const getFivePercentValueOfCurrentExchangeRate = 0.1 * finalRate
+        const getMinimumFivePercentOfCurrentExchangeRate =
+          finalRate - getFivePercentValueOfCurrentExchangeRate
+        cy.log('Mimnimum is')
+        cy.log(getMinimumFivePercentOfCurrentExchangeRate)
+        const getMaximumFivePercentOfCurrentExchangeRate =
+          finalRate + getFivePercentValueOfCurrentExchangeRate
+        cy.log('Maximum is')
+        cy.log(getMaximumFivePercentOfCurrentExchangeRate)
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js:24]:
```suggestion
        cy.log(`Maximum is: ${getMaximumFivePercentOfCurrentExchangeRate}`)
```

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js [hunk 2]
```diff
@@ -38,7 +72,7 @@ describe('QATEST-98789 - Transfer to crypto accounts and QATEST-98794 View Crypt
     cy.c_login({ user: 'walletloginEmail' })
   })
 
-  it('should be able to perform transfer from crypto account', () => {
+  it.only('should be able to perform transfer from crypto account', () => {
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js:75]: Is this `only` here intentional?

**Review by abuzar-deriv — COMMENTED** [review]
Some minor suggestions but otherwise looks good.

File: cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js [hunk 3]
```diff
@@ -1,16 +1,46 @@
...
+          finalRate - getFivePercentValueOfCurrentExchangeRate
+        cy.log(`Mimnimum is: ${getMinimumFivePercentOfCurrentExchangeRate}`)
+        const getMaximumFivePercentOfCurrentExchangeRate =
+          finalRate + getFivePercentValueOfCurrentExchangeRate
+        cy.log(`Maximum is: ${getMaximumFivePercentOfCurrentExchangeRate}`)
+        const TrasnferValue = parseFloat(val.split(' ')[0])
+        expect(TrasnferValue).to.be.greaterThan(
+          getMinimumFivePercentOfCurrentExchangeRate
+        )
+        expect(TrasnferValue).to.be.gte(
```
Comment (abuzar-deriv) [inline cypress/e2e/smoke/Wallets/cryptoTransactions.cy.js:25]: Can you please fix the typo here?

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**